components/accelerator/nvidia/hw-slowdown/component.go

```diff
@@ -82,7 +142,7 @@ func (c *component) Events(ctx context.Context, since time.Time) ([]components.E
 		convertedEvents = append(convertedEvents, components.Event{
 			Time:    metav1.Time{Time: time.Unix(event.UnixSeconds, 0).UTC()},
 			Name:    EventNameHWSlowdown,
-			Type:    components.EventTypeInfo,
+			Type:    components.EventTypeWarning,
```

Comment:
@cardyok @xiang90 Now,

1. all hw slowdown events are warning
2. states are healthy as long as for the last 10 minutes, there are <5 hw slowdown warning events
3. states become unhealthy for the last 10 minutes, there are >=5 hw slowdown warning events

Reply:
can we make it a ratio? like 60%? do not use abs value

Reply:
@xiang90 @cardyok Now it's checking the per-minute frequency 60%.

For the last 10 minutes, 6 events make 60% ratio, thus unhealthy.
For the last 10 minutes, 3 events make 30% ratio, thus healthy.
If we use 100 minutes, 60 events make 60% ratio, thus unhealthy.